unit-tests-jest/numbers.ts

```diff
@@ -33,7 +38,11 @@ export function multiplyBy(
 ): Record<string, unknown> {
   const result: Record<string, unknown> = {};
   Object.entries(obj).forEach(([key, value]) => {
-    result[key] = Number(value) * multiplier;
+    if (typeof value === 'string') {
```

Comment:
You should actually check the other way around. Only do the multiplication if it's a number.